.env.example

```diff
@@ -0,0 +1 @@
+NEXT_PUBLIC_BACKEND_URL = http://localhost:3001
```

Comment:
write a note in readme on what this variable refers to

Reply:
Added in Readme